File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.js
function solution(A,B){
    // 1. A, B 정렬하기
    A.sort((a,b) => a-b); // 오름차순 정렬
    B.sort((a,b) => b-a); // 내림차순 정렬
    
    // 2. 두 수 뽑아서 더하기
    let answer = 0;
    for(let i=0; i<A.length; i++) {
        answer += A[i] * B[i];
    }
    return answer;
}

File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.java
import java.io.*;
import java.util.*;

class Solution
{
    public int solution(int []A, int []B)
    {
        // 1. A, B 정렬
        Arrays.sort(A);
        Arrays.sort(B);
        
        // 2. 두 수를 뽑아 더하기
        int answer = 0;
        for(int i=0; i<A.length; i++) {
            answer += A[i] * B[B.length-1-i]; // (작은 수 * 큰 수) 짝으로 곱해서 더하기
        }
        return answer;
    }
}